exporter/collector.go

```diff
@@ -100,6 +104,9 @@ func (mc JSONMetricCollector) Collect(ch chan<- prometheus.Metric) {
 				level.Error(mc.Logger).Log("msg", "Failed to convert extracted objects to json", "err", err, "metric", m.Desc)
 				continue
 			}
+		default:
```

Comment:
I am wondering if we can still keep the previous behavior if the type is not specified explicitly? Inferring it from `ValueJSONPath`

Reply:
The default in the config parser is `ValueScrape` if it's not specified. In theory, we should never hit this default, but wanted it here as a sanity check.